### PR TITLE
add what is a wallet and what is a network page

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- feat: Add UI for `what-is-a-wallet` and `what-is-a-network`

### CODE REVIEW

File: packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    text: 'A wallet lets you store, send and receive digital assets like cryptocurrencies and NFTs.'
+  }
+]
+
+@customElement('w3m-what-is-a-network-view')
+export class W3mWhatIsANetworkView extends LitElement {
+  public static styles = styles
+
+  // -- Render -------------------------------------------- //
+
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts:31]:
```suggestion
```

File: packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  {
+    images: ['noun', 'defiAlt', 'dao'],
+    title: 'Designed for different uses',
+    text: 'A wallet lets you store, send and receive digital assets like cryptocurrencies and NFTs.'
+  }
+]
+
+@customElement('w3m-what-is-a-network-view')
+export class W3mWhatIsANetworkView extends LitElement {
+  public static styles = styles
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts:28]: Add style reset

File: packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import { VisualType } from '@web3modal/ui/src/utils/TypesUtil'
+import { RouterController } from '@web3modal/core'
+
+type Data = {
+  images: VisualType[]
+  title: string
+  text: string
+}
+
+const data: Data[] = [
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts:13]: `Data` type is probably not necessary here. If it needs to be here to satisfy something further down below, you can add `as const` at the end of the data array.

File: packages/scaffold-html/src/views/w3m-what-is-a-wallet-view/index.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    text: 'With your wallet, you can explore and interact with DeFi, NFTs, DAOs, and much more.'
+  }
+]
+
+@customElement('w3m-what-is-a-wallet-view')
+export class W3mWhatIsAWalletView extends LitElement {
+  public static styles = styles
+
+  // -- Render -------------------------------------------- //
+
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-wallet-view/index.ts:37]:
```suggestion
```

File: packages/scaffold-html/src/views/w3m-what-is-a-wallet-view/index.ts [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import { VisualType } from '@web3modal/ui/src/utils/TypesUtil'
+import { RouterController } from '@web3modal/core'
+
+type Data = {
+  images: VisualType[]
+  title: string
+  text: string
+}
+
+const data: Data[] = [
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-wallet-view/index.ts:14]: Same comment about data type as above.

File: packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+@customElement('w3m-what-is-a-network-view')
+export class W3mWhatIsANetworkView extends LitElement {
+  public static styles = styles
+
+  // -- Render -------------------------------------------- //
+
+  public render() {
+    return html`
+      <wui-flex
```
Comment (0xAsimetriq) [inline packages/scaffold-html/src/views/w3m-what-is-a-network-view/index.ts:34]: Looks like we are repeating same bit of code here as with another help view, perhaps we can make a partial i.e. `w3m-help-widget` that takes in params like data array and onLearnMore function? We'll likely need another optional param like `onAction` for that "Get wallet" button latter on.

**Comment by svenvoskamp** [comment]
@0xAsimetriq 

I've added a `w3m-help-widget` partial. It only takes in the data for the visuals. Button stuff stays in the specific view component. Makes most sense to me.

**Review by 0xAsimetriq — APPROVED** [review]

**Pull request merged**